### PR TITLE
Load synth presets into kit rows

### PR DESCRIPTION
**Pull request by m-m-adams**
Pseudo draft - it's working but hacky, I need to do a review and figure out what can get cut back now

### CODE REVIEW

**Comment by m-m-adams** [comment]
I think it's good now

File: src/deluge/gui/ui/load/load_instrument_preset_ui.cpp [hunk 1]
```diff
@@ -938,6 +947,37 @@ int32_t LoadInstrumentPresetUI::performLoad(bool doClone) {
 	return NO_ERROR;
 }
 
+int32_t LoadInstrumentPresetUI::performLoadSynthToKit() {
+	FileItem* currentFileItem = getCurrentFileItem();
+	ParamManager* parammanager;
```
Comment (seangoodvibes) [inline src/deluge/gui/ui/load/load_instrument_preset_ui.cpp:952]: can you change parammanager to paramManager :)?

File: src/deluge/gui/ui/load/load_instrument_preset_ui.cpp [hunk 2]
```diff
@@ -938,6 +947,37 @@ int32_t LoadInstrumentPresetUI::performLoad(bool doClone) {
...
+	ModelStackWithTimelineCounter* modelStack = currentSong->setupModelStackWithCurrentClip(modelStackMemory);
+
+	int32_t error = storageManager.loadSynthToDrum(currentSong, instrumentClipToLoadFor, false, &soundDrumToReplace,
+	                                               &currentFileItem->filePointer, &enteredText, &currentDir);
+	//kitToLoadFor->addDrum(soundDrumToReplace);
+	soundDrumToReplace->loadAllAudioFiles(true);
+	//soundDrumToReplace->name.set(getCurrentFilenameWithoutExtension());
+	getCurrentFilenameWithoutExtension(&soundDrumToReplace->name);
+	ModelStackWithNoteRow* modelStackWithNoteRow = modelStack->addNoteRow(noteRowIndex, noteRow);
+	parammanager = currentSong->getBackedUpParamManagerPreferablyWithClip(soundDrumToReplace, instrumentClipToLoadFor);
```
Comment (seangoodvibes) [inline src/deluge/gui/ui/load/load_instrument_preset_ui.cpp:973]: parammanager >> paramManager

**Review by seangoodvibes — CHANGES_REQUESTED** [review]
Just one minor formatting change and then you're good to go.

**Review by seangoodvibes — APPROVED** [review]
Looks good. Approved!

**Comment by soymonitus** [comment]
This needs to be rolled back i think! Loading kits (LOAD + Kit) no longer works, it shows the synth selector. Loading a synth also doesn't work, the synth doesn't change. Also when holding a row in Song mode and changing its type makes the Deluge hang

**Comment by soymonitus** [comment]
Reverted this commit to test again and confirmed everything works good without this change applied

**Pull request merged**